src/Schema/Comparator.php

```diff
@@ -44,6 +48,32 @@ public function __construct(?AbstractPlatform $platform = null)
         $this->platform = $platform;
     }
 
+    /**
+     * @param list<mixed> $args
```

Comment:
FWIW, on PHP 8 this is not a list anymore.

Reply:
See https://github.com/doctrine/dbal/pull/5220#discussion_r795079047 :slightly_smiling_face: